### PR TITLE
MC-635 adding operation resetAgeStatistics

### PR DESCRIPTION
**Pull request by erosb**
Adds a simple operation for resetting the minAge, maxAge and averageAge metrics of a queue. 

Question to the reviewers: what should be the `since` version of new operations?

Platform-side PR: https://github.com/hazelcast/hazelcast/pull/19625

### CODE REVIEW

**Comment by mdumandag** [comment]
This operation will be used by the management center only, right? I feel like `MC` service would be a better place for it, but not %100 sure. @sancar WDYT?

**Comment by sancar** [comment]
https://github.com/hazelcast/hazelcast-client-protocol/blob/master/protocol-definitions/MC.yaml#L157
It looks like there are similar usages on the MC.yaml 
I would also prefer to keep them MC.yaml.

**Comment by erosb** [comment]
Hello @sancar @mdumandag I moved the new operation to MC.yaml.
One more question: it is tagged with `since: 2.4` . Is it a correct version number?

**Comment by mdumandag** [comment]
@erosb thanks. Yes, that is the correct version number

File: protocol-definitions/MC.yaml [hunk 1]
```diff
@@ -845,3 +845,17 @@ methods:
       retryable: false
       partitionIdentifier: -1
     response: {}
+  - id: 33
+    name: resetAgeStatistics
```
Comment (mdumandag) [inline protocol-definitions/MC.yaml:849]: Maybe rename it to `resetQueueAgeStatistics` as this is under the new namespace that is not related to the queue?

**Comment by erosb** [comment]
run-lab-run

**Review by mdumandag — APPROVED** [review]
